[DOCS] Remove all mentions of profilers in public docs (#8825)
great-expectations/great_expectations: docs/docusaurus/docs/terms/data_asset.md

The specifics of a filesystem Data Asset are defined by the parameters provided when it is created. With a SQL Data Asset, you can also add splitters after you have initially created the Data Asset.

> You can define multiple Data Assets built from the same underlying source data system to support different workflows such as interactive exploration and creation of <TechnicalTag relative="../" tag="expectation" text="Expectations" /> and ongoing <TechnicalTag relative="../" tag="validation" text="Validation" /> through <TechnicalTag relative="../" tag="checkpoint" text="Checkpoints" />.

Reviewer: ```suggestion
You can define multiple Data Assets from the same Data Source to support different workflows. For example,  interactive exploration and creation of <TechnicalTag relative="../" tag="expectation" text="Expectations" /> and  continuous<TechnicalTag relative="../" tag="validation" text="Validations" /> with <TechnicalTag relative="../" tag="checkpoint" text="Checkpoints" />.
```
